packages/opds-web-client/src/components/BookDetails.tsx

```diff
@@ -17,43 +17,43 @@ export default class BookDetails<P extends BookDetailsProps> extends Book<P> {
           <div className="cover">
             <BookCover book={this.props.book} />
           </div>
-          <div className="header">
+          <div className="details">
             <h1 className="title">{this.props.book.title}</h1>
             {
               this.props.book.subtitle &&
-              <h3 className="subtitle">{ this.props.book.subtitle }</h3>
+              <p className="subtitle">{ this.props.book.subtitle }</p>
             }
             {
               this.props.book.series && this.props.book.series.name &&
-              <h3 className="series">{ this.props.book.series.name }</h3>
+              <p className="series">{ this.props.book.series.name }</p>
             }
             {
-              this.props.book.authors && this.props.book.authors.length &&
-              <h2 className="authors">By {this.props.book.authors.join(", ")}</h2>
+              this.props.book.authors && this.props.book.authors.length > 0 &&
+              <p className="authors">By {this.props.book.authors.join(", ")}</p>
             }
             {
-              this.props.book.contributors && this.props.book.contributors.length &&
-              <h2 className="contributors">Contributors: {this.props.book.contributors.join(", ")}</h2>
+              this.props.book.contributors && this.props.book.contributors.length > 0 &&
+              <p className="contributors">Contributors: {this.props.book.contributors.join(", ")}</p>
             }
-            <div className="fields" lang="en">
+            <ul className="fields" lang="en">
```

Comment:
This being a list seems a bit weird since these are a bunch of unrelated items, but if you all think it makes sense that's fine.

Reply:
They were all grouped before and it's a list of the book's detail, so not exactly unrelated.